File: find_pivot/find_pivot.c
int		find_pivot(int *arr, int n)
{
	int i;
	int sum = 0;
	int left = 0;

	i = 0;
	while (i < n)
	{
		sum += arr[i];
		i++;
	}
	i = 0;
	while (i < n)
	{
		sum -= arr[i];
		if (sum == left)
			return (i);
		left += arr[i];
		i++;
	}
	return (-1);
}
#include <stdio.h>
int main()
{
	int arr[5] = {1, 100, 0, 0, 1};

	printf("%i\n", find_pivot(arr, 5));
}

File: 0/find_pivot/find_pivot.py
def find_pivot(arr, arr_len):
    i = 0
    sum = 0
    left = 0
    while i < arr_len:
        sum += arr[i]
        i += 1
    i = 0
    while i < arr_len:
        sum -= arr[i]
        if sum == left:
            return i
        left += arr[i]
        i += 1
    return (-1)

arr = [1,3,1]
print(find_pivot(arr, 3))